test/e2e/policyrecommendation_test.go

```diff
@@ -174,7 +175,7 @@ func testPolicyRecommendationRun(t *testing.T, data *TestData) {
 	stdout, jobId, err := runJob(t, data)
```

Comment:
```suggestion
	stdout, jobName, err := runJob(t, data)
```
Same for other jobID